src/insexpand.c

```diff
@@ -3958,14 +3958,13 @@ fuzzy_longest_match(void)
     }
 
     leader = ins_compl_leader();
-    if (leader != NULL)
-	leader_len = STRLEN(leader);
+    leader_len = ins_compl_leader_len();
 
     // skip non-consecutive prefixes
-    if (STRNCMP(prefix, leader, leader_len) != 0)
+    if (leader_len > 0 && STRNCMP(prefix, leader, leader_len) != 0)
```

Comment:
This may fix a Coverity warning for NULL dereference.

Reply:
Yes, thanks